packages/components/src/chart/index.js

```diff
@@ -272,13 +272,13 @@ class Chart extends Component {
 
 		switch ( valueType ) {
 			case 'average':
-				yFormat = '.0f';
+				yFormat = ',.0f';
```

Comment:
I'm not really savvy with d3 formatting, but does this somehow pull in the user's locale and format properly?

Reply:
Yes, it does. This comma here will be replaced by a dot or any other symbol depending on the user locale format.

On top of the file we specify the default format:

https://github.com/woocommerce/wc-admin/blob/master/packages/components/src/chart/index.js#L29-L34

However, I tried experimenting with other locales and it looks like we don't honor them, and we are always using the default format. I filled a bug for that:  #1322.